src/generated/github-schema-loader.js

```diff
@@ -0,0 +1,6 @@
+import { schema } from '@octokit/graphql-schema';
+
+import('@graphql-codegen/typescript');
+import('@graphql-codegen/typescript-operations');
```

Comment:
I guess you want static imports (`import ...`) instead of dynamic imports (`import(...)`) here (these are asynchronous, and you're not using `await` with them)

Reply:
```js
import '@graphql-codegen/typescript';
import '@graphql-codegen/typescript-operations';
import { schema } from '@octokit/graphql-schema';
```

Changed imports in this commit
[af86570](https://github.com/upleveled/next-portfolio-dev/pull/107/commits/af865707fd5d4286d624392c9eb1c554a968191e)